File: Chef On Island/sample_solution.cpp
#include <iostream>
using namespace std;

int main() {
	int o;
	cin>>o;
	while(o--)
	{
	    int x,y,x1,y1,d;
	    cin>>x>>y>>x1>>y1>>d;
	    x=x/x1;
	    y=y/y1;
	    if(min(x,y)>=d)
	    {
	        cout<<"yes\n";
	        
	    }
	    else{
	        cout<<"no\n";
	    }
	    
	}
	return 0;
}

File: Chef On Island/sample_solution.py
import math
n=int(input())
for i in range(n):
    l=[]
    a=list(map(int,input().split()))
    b=a[0]/a[-3]
    c=a[1]/a[-2]
    #print(b)
    #print(c)
    l.append(b)
    l.append(c)
    d=min(l)
    if a[-1]<=d:
        print("Yes")
    else:
        print("No")
